rockcraft.yaml

```
@@ -40,6 +42,8 @@ parts:
         stage-packages:
             - percona-server-mongodb
             - util-linux
+            - mongodb-mongosh
+            - mongodb-mongosh-shared-openssl3
```

I think we can install it from the Percona repo
```suggestion
            - percona-mongodb-mongosh
```
otherwise, we will receive MongoDB itself, and MongoDB shell received from different source codes.

---

Looking at the percona repo and testing, there is no `percona-mongodb-mongosh` or similar package available.  AFAIK adding the repo from mongodb with mongosh does not install the server components

---

works fine for me
```
root@c00455f6e4ef:/# percona-release enable psmdb-50 release
* Enabling the Percona Server for MongoDB 5.0 repository
<*> All done!
==> Please run "apt-get update" to apply changes
root@c00455f6e4ef:/# apt update
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://security.ubuntu.com/ubuntu jammy-security InRelease
Get:3 http://repo.percona.com/psmdb-50/apt jammy InRelease [15.7 kB]
Hit:4 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:5 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Get:6 http://repo.percona.com/psmdb-50/apt jammy/main Sources [2019 B]
Get:7 http://repo.percona.com/psmdb-50/apt jammy/main amd64 Packages [7323 B]
Fetched 25.0 kB in 1s (41.6 kB/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
root@c00455f6e4ef:/# apt install percona-server-mongodb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libsasl2-modules-gssapi-mit percona-server-mongodb-mongos percona-server-mongodb-server percona-server-mongodb-shell percona-server-mongodb-tools
The following NEW packages will be installed:
  libsasl2-modules-gssapi-mit percona-server-mongodb percona-server-mongodb-mongos percona-server-mongodb-server percona-server-mongodb-shell percona-server-mongodb-tools
0 upgraded, 6 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
Need to get 131 MB of archives.
After this operation, 340 MB of additional disk space will be used.
Do you want to continue? [Y/n]
Get:3 http://repo.percona.com/psmdb-50/apt jammy/main amd64 percona-server-mongodb-shell amd64 5.0.14-12.jammy [18.0 MB]
```

---

Installing `percona-server-mongodb` isnt the issue. The problem is that I need `mongosh` as part of the image which is not provided as part of that repository.

```
root@mongo:~# apt update
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Get:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [114 kB]                                      
Get:3 http://repo.percona.com/percona/apt jammy InRelease [15.7 kB]                                          
Get:4 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]                                    
Get:5 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [99.8 kB]                                   
Hit:6 http://repo.percona.com/psmdb-50/apt jammy InRelease                                                   
Hit:7 https://ppa.launchpadcontent.net/data-platform/mongodb/ubuntu jammy InRelease         
Get:8 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [831 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [566 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [786 kB]
Fetched 2523 kB in 1s (1934 kB/s)                        
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
36 packages can be upgraded. Run 'apt list --upgradable' to see them.
root@mongo:~# apt install percona-mongodb-mongosh -y
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package percona-mongodb-mongosh
```

---

@delgod I agree that we should install percona mongodb shell and not the official mongodb community one.

But when I followed your instructions (and the [official percona install instructions](https://docs.percona.com/percona-server-for-mongodb/5.0/install/apt.html)) I ran `mongosh` and ran into the same error as @jardon :
```
ubuntu@ip-172-31-3-19:~$ mongosh
mongosh: command not found
```
and 
```
ubuntu@ip-172-31-3-19:~$ ls /usr/bin/ | grep mongo
mongo
mongobridge
mongod
mongodump
mongoexport
mongofiles
mongoimport
mongorestore
mongos
mongostat
mongotop
percona-server-mongodb-enable-auth.sh
percona-server-mongodb-helper.sh
```
doesn't show `mongosh` does `ls /usr/bin/ | grep mongo` show `mongosh` for you? Or does `which mongosh` show any useful information?